doc/api/stream.md

```diff
@@ -661,19 +664,19 @@ pass.unpipe(writable);
 
 pass.on('data', (chunk) => { console.log(chunk.toString()); });
 pass.write('ok'); // will not emit 'data'
-pass.resume(); // must be called to make 'data' being emitted
+pass.resume(); // must be called to make 'data' be emitted
```

Comment:
Hm, wdyt of: 'must be called to make stream emit `data`'?

Reply:
Can do.  However, I have a concern about implying determinism.  What about "now the stream can emit 'data' events"?  Correct me if I am wrong, but I think calling `resume()` does not mean that a `data` event will necessarily be emitted, depending on the state of the stream.

Reply:
It will be emitted after resume if there is data in the stream and we have just added it there via the above `write` call. Therefore it should be fine with being deterministic here.

Reply:
Rebased, and both of your changes applied, in 0bb51b0c96.  Thanks!